definitions/npm/backbone_v1.3.x/flow_>=v0.25.x/backbone_v1.3.x.js.flow

```diff
@@ -0,0 +1,68 @@
+// @flow
+import typeof Underscore from 'underscore';
+
+declare module 'backbone' {
+   declare var $: any;
```

Comment:
Since we still don't have a jQuery definition or a way to deal with dependencies between library definitions, I wasn't sure the best way to `Backone.$` or `Backbone._` What should I do now for these, leave `any`?